testsol.sh

```diff
@@ -0,0 +1,56 @@
+function esolc() {
+    file=$1
+    echo $file
+    shift
+    cabal run sol-core -- -f $file $*
+}
+
+function testsol() {
+    file=$1
+    echo $file
+    shift
+    cabal run sol-core -- -f $file $* && \
+	cabal exec yule -- output.core -w -O > /dev/null && \
+    forge script Output.sol | grep RESULT
+}
+
+function testspec() {
+    file=$1
+    echo $file
+    shift
+    cabal exec sol-core -- -f $file --debug-spec --dump-spec $*
+#    cabal run yule -- output.core -O && \
+#    forge script Output.sol
```

Comment:
Is this supposed to be commented?

Reply:
Actually `testspec` is just to debug the cases where specialisation fails. Hence they are usually not executed.